decidim/decidim
# Redesign: fix responsiveness on nav-bar admin

#### :tophat: What? Why?
This fixes an issue with the nav-bar that allows users on mobile on smaller screens to navigate and use features available there with no responsive issues.

#### :pushpin: Related Issues

- Related to #11956 
- Fixes #11992

#### Testing
1. Login as a admin and head over to edit (admin panel)
2. Find a module like Processes and access one
3. Head over to a component like Accountability 
4. Make your screen smaller like mobile dimensions 
5. See the nav bar options be more accessible

### :camera: Screenshots
![Screenshot 2023-11-21 at 12 03 01](https://github.com/decidim/decidim/assets/101816158/a0cd8cd9-b765-448e-ae3f-6e286d442e90)


:hearts: Thank you!
